.github/workflows/e2e.yaml

```diff
@@ -74,7 +68,35 @@ jobs:
           distribution: "temurin"
           java-version: ${{ matrix.java }}
           cache: "maven"
-      - name: Run ${{ matrix.broker }} Platform ${{ matrix.kafka }} E2E Tests
+      - name: Run Confluent Kafka Platform ${{ matrix.kafka }} E2E Tests
+        run: |
+          echo "Running integration tests with Docker Compose file: ${COMPOSE_FILE}"
+          docker compose -f ${COMPOSE_FILE} pull
+          mvn --batch-mode --no-transfer-progress --errors --update-snapshots clean verify -Drevision=$(git describe --tags --always)
+
+  e2e-redpanda:
+    strategy:
+      matrix:
+        os: [ubuntu-22.04]
+        java: [ "11", "17" ]
+        kafka: ["3.7.1","3.8.0"]
+        redpanda: ["24.2.1"]
```

Comment:
Sorry, I should have said `v24.2.1`, my fault: https://hub.docker.com/layers/redpandadata/redpanda/v24.2.1/images/sha256-be577643be8c4f08063bf2c680175478e0bbae0bde9d24cf61e05ec777d9c0aa?context=explore

Reply:
The bug :rofl: 
```
$ sed -e 's/v24.1.8/24.2.1/g' --in-place .github/workflows/e2e.yaml e2e/docker_compose_redpanda.yaml
```